src/opmath.cpp

```diff
@@ -84,10 +84,8 @@ int32_t op_shift_right_unsigned(int32_t value, int32_t amount)
 	// Repeat the easy cases here to avoid INT_MIN funny business
 	if (amount == 0)
 		return value;
-	if (value == 0 || amount <= -32)
+	if (value == 0 || amount < -31 || amount > 31)
 		return 0;
-	if (amount > 31)
-		return (value < 0) ? -1 : 0;
```

Comment:
Copy-paste error, I'm assuming? That would mean this is a genuine bugfix!

Reply:
Yep!

Reply:
We may want to add that to the release notes, then:

- `-1 >>> 32` returned `$FFFFFFFF`, not 0.